src/Arcus.Security.Providers.AzureKeyVault/KeyVaultSecretProvider.cs

```diff
@@ -77,6 +84,44 @@ public KeyVaultSecretProvider(IKeyVaultAuthentication authentication, IKeyVaultC
                 "Requires the Azure Key Vault host to be in the right format, see https://docs.microsoft.com/en-us/azure/key-vault/general/about-keys-secrets-certificates#objects-identifiers-and-versioning");
 
             _authentication = authentication;
+            _isUsingAzureSdk = false;
```

Comment:
Let's create an issue to remove our deprecated authentication in v2

Reply:
#202